scipy/stats/_bootstrap.py

```diff
@@ -208,6 +225,9 @@ def bootstrap(data, statistic, *, vectorized=True, axis=0,
         If `vectorized` is set ``False``, `statistic` will not be passed
         keyword argument `axis`, and is assumed to calculate the statistic
         only for 1D samples.
+    paired : bool, optional
```

Comment:
```suggestion
    paired : bool, default: ``False``
```
I meant to do at least this one. I'll do these locally.